playbooks/roles/ocp-fio/defaults/main.yml

```diff
@@ -0,0 +1,20 @@
+---
+
+fio_install_operator: true
+fio_catalogsource_image: ""            # brew.registry.redhat.io/rh-osbs/iib:276833
```

Comment:
Should the value be "redhat-operators" as per docs?

Reply:
Its default value should be empty, it should use default catalog source . I will update the doc accordingly